### DISCUSSION TITLE
Automaticaly record Tablet GPS location of data collectors while they edit a point feature

### DISCUSSION THREAD

**Discussion by SimonGuillouet** [opengisch/QField#4920] [Q&A]
Hello all :)

I am seting up a field survey to inform some points. The goal is to ask to data collectors to go to a series of points and to register some basic attribute information on those points. 

I would like to monitor the data collector by capturing their GPS position while they edit a point. This is not very complicated if they create a new point as the default expression "x(@gnss_coordinate)" and "y(@gnss_coordinate)" is able to do the job. **HOWEVER**, when data collectors just edit an existing point, the XY GPS location of the tablet are not recorded and the field is left blanck even thought I ticked the box to "Update the default value" in the attribute form...

How can I "force" QFIELD to record this information ? Is their a different expression or a default setting to customize ?

Thanks in advance, best regard and... happy new year :) !

**Comment by VxTedxV** [comment]
May be aware of field format and its length...

**Reply by SimonGuillouet** [reply]
Thanks for the answer VxTedxV! But as it works when I create a new point I think the problem is coming from somewhere else.
